TYPO3.Flow/Classes/TYPO3/Flow/Command/HelpCommandController.php

```diff
@@ -29,30 +29,58 @@
 class HelpCommandController extends CommandController
 {
     /**
-     * @Flow\Inject
      * @var PackageManagerInterface
      */
     protected $packageManager;
 
     /**
-     * @Flow\Inject
      * @var Bootstrap
      */
     protected $bootstrap;
 
     /**
-     * @Flow\InjectConfiguration(path = "core.applicationPackageKey")
      * @var string
      */
     protected $applicationPackageKey;
 
     /**
-     * @Flow\Inject
      * @var CommandManager
      */
     protected $commandManager;
 
     /**
+     * @param Bootstrap $bootstrap
```

Comment:
<nitpick>Not sure about our current policy regarding `@return void` annotations, but it's missing here ;)</nitpick>